### ISSUE TITLE
Enable model / call timeouts

### ISSUE THREAD

**Issue by krrishdholakia** [BerriAI/litellm#2]

**Comment by ishaan-jaff** [comment]
what is the expected behavior here ?

**Comment by ghost** [comment]
Azure hangs so: 

```
response = completion(model=model, messages=messages, azure=azure, set_timeout=45)
```

where `set_timeout` takes in a time input in seconds. 

**Nice to have**: It also does feel weird to pass in `azure = True`. The way I modified this was by setting it to be `llm_provider="azure"`

**Comment by ghost** [comment]
implemented in v0.1.2
